addons/l10n_in_tds_tcs_warning/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': 'Indian - TDS, TCS Warning',
```

Comment:
We initially did it in `l10n_in` but since it's only applicable to particular business and to reduce a bit of complexity. Decided to keep the module separate. Rest it's better have this discussion with @jva-odoo

Reply:
But it could all go into l10n_in_withholding?

Reply:
@bhra-odoo let's move to l10n_in_with_holding

Reply:
@jco-odoo but if we move code to l10n_in_withholding
We need to figure out a way to update the TCS taxes (existing `l10n_in`) and withholding introduces TDS taxes. Can you suggest a way where we can update the existing TCS taxes and create new TDS taxes in the same file `account.tax-in.csv`, because we don't want to update other fields of TCS only want to update `l10n_in_section_id`.
and I guess `_parse_csv` won't allow two different files for `account.tax` :/

Reply:
Of course, you could copy the taxes entirely... (both l10n_in and l10n_in_withholding)  Or only have them in l10n_in_withholding anyways.  (that way if they need withholding, they install the module)

Reply:
Well okay, we do it that way, because I was thinking to avoid the duplication of data for TCS and just add section_id to it. But seems like there is no other way for it